Stanford/JumpstartEngineering/Install/Menu/JSEMenuRedirects.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @file
+ * Create redirects.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Menu;
+/**
+ *
+ */
+class JSEMenuRedirects extends \AbstractInstallTask {
+
+  /**
+   * Create menu redirects.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    // Create redirects.
+    $redirects = array(
+      'research' => 'research/overview',
+      'people' => 'people/all/grid/grouped',
+      'news' => 'news/recent-news',
+      'events' => 'events/upcoming-events',
+      'about' => 'about/mission',
+      'resources' => 'resources/overview',
+    );
+    foreach ($redirects as $source => $dest) {
+      $redirect = new \stdClass();
```

Comment:
Wanted to get a chance to test it locally first :). I just wrote that utility class at like 4:55PM yesterday.